client/src/assets/scss/foundation/_settings.scss

```diff
@@ -872,7 +872,7 @@ $tooltip-radius: $global-radius;
 // 55. Top Bar
 // -----------
 
-$topbar-padding: 0.5rem;
+$topbar-padding: 0.2rem;
```

Comment:
you should prefer editing the `main.scss` file over changing the Foundation styles directly. That way it is easier to trace what styles you have implemented vs ones from Foundation